tests/test_embedding.py

```diff
@@ -37,6 +37,17 @@ def test_embeddings_with_norms_oov():
     assert embeds.embedding_with_norm("Something out of vocabulary") is None
 
 
+def test_indexing():
+    embeds = finalfusion.Embeddings(
+        "tests/embeddings.fifu")
+    assert embeds["one"] is not None
+    try:
```

Comment:
Use:

```
with pytest.raises(KeyError):
    embeds["Something out of vocabulary"]
```

Also, with #40, leading the embeddings here is not necessary anymore ;).